Automattic/jetpack
# [Super Cache] Fix undefined variable

Fixes #25469.

## Proposed changes:

* Fix undefined PHP variable when trying to delete the cache folder.

### Other information:


## Jetpack product discussion


n/a

## Does this pull request change what data or activity we track or use?


no

## Testing instructions:






### Pre-requisites
Make sure this version of WP Super Cache is running and is enabled on the test website. wp debug should also be enabled. Make sure that there's also cache generated `wp-content/cache/supercache` should contain a folder with the domain of the test website. If it's not present, make sure cache is enabled and visit the home page while not logged-in to generate it.

### Test to make sure no PHP warning appears
* Change the permission of the website cache folder to something like `644` to make sure the web user can't delete it;
* Go to Super Cache settings and clear the cache;
* Check the log - there should be no warning that `$donotdelete` is not defined.
